openid-4-verifiable-presentations-1_0.md

```diff
@@ -320,6 +320,25 @@ The following is a non-normative example of a transaction data content, after ba
 }
 ```
 
+`verifier_attestations`:
+: OPTIONAL. An array of signed objects used by the Verifier to convey attested information relevant to the Credential Request. These attestations MAY include Verifier metadata, policies, trust status, or authorizations. Attestations are intended to support authorization decisions, inform Wallet policy enforcement, or enrich the End-User consent dialog. Each object has the following structure:
+
+    * `type`: REQUIRED. A string that identifies the type of the attestation and how it is encoded. Ecosystems SHOULD use collision-resistant identifiers.
```

Comment:
I miss the format/type differentiation. The way it is defined now, the type determines both (which is different from all other places in OID4VC)

Reply:
I don't think that we need the `format` field like it is used in DCQL. Each attestation object hast the three mentioned fields, but the value of `type` will help you to understand the value of data and how to process it. E.g. when the value is `eudi_registration_certificate` and it's a known value to the Wallet, it will know that the `data` field of type string, including a JWT with a known structure.
There was no demand from others that a type of attestation can be passed in different formats (like JWT or JWS JSON serialization). If this is a demand, the eco system should include the encoding information into the value of `type` like `eudi_registration_certificate+jwt`.

Reply:
I think Torsten's suggestion makes sense - use a type to enable decoding, then use format-specific mechanisms to transport the purpose of the attestation.

Reply:
We agreed it's better to define the format for decoding than the type. This has the disadvantage that the Wallet needs to decode all attestations to get to type (decoding the sd jwt vc and then looking at the vct value). Since this is only decoding and not validating, the required process is affordable. This reduces the risk that there is a difference between an attribute on the root level of the attestation and the actual passed object